packages/react/src/lib/felaDisableAnimationsPlugin.ts

```diff
@@ -0,0 +1,26 @@
+import * as _ from 'lodash'
+
+export default () => {
```

Comment:
lets document behavior of this plugin in comment - specifically, the following facts
- it is triggered by the styles' prop value (in contrast to many other Fela plugins)
- what is the name of the prop value

Reply:
Added, please take a look. We should add this for the other plugins that potentially can cause infinite recursion.